sunpy/physics/limb_darkening.py

```diff
@@ -82,15 +74,14 @@ def darklimb_v(wavelength):
 
     a = np.array((av, bv, cv, dv, ev, fv))
 
-    wavelength = wavelength.to_value()
-    wavelength_series = np.array([wavelength ** n for n in range(len(a))])
+    wavelength_series = np.array([wavelength.value ** n for n in range(len(a))])
```

Comment:
Do we need to `.value` here or can we do this whole function with units?

Reply:
It would only work with scalars. Otherwise throws an error.